src/Components/Artist/MarketInsights/MarketInsights.tsx

```diff
@@ -3,15 +3,15 @@ import * as React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components"
 
-import colors from "../../../Assets/Colors"
-import { Fonts } from "../../Publishing/Fonts"
+import colors from "Assets/Colors"
+import { unica } from "Assets/Fonts"
```

Comment:
We should make 100% sure that absolute paths are cross-app compatible; maybe there's something easy we can do if not?

Can you try to run this while linked with force?

Reply:
I did get some errors in force but I'm not 100% sure it's related to the use of absolute paths. Will confirm, if anything i'll revert back to relative paths

Reply:
@l2succes - how did this end up working in force?

Reply:
@damassi Tested it locally on two machines and it works great. There was an issue with jest not resolving the path aliases but I fixed it

Reply:
👍